app/notify_admin.go

```diff
@@ -116,51 +118,90 @@ func (a *App) SendNotifyAdminPosts(c *request.Context, workspaceName string, cur
 
 	userBasedData := a.groupNotifyAdminByUser(data)
```

Comment:
I think you need two different user based data variables, one specific to plugin installs, and one specific to plan notifications. Otherwise, when the job runs and there is some of each type of send, it will overcount how many users requested a feature. This may require adding a separate user grouping method for grouping by user & plugin install type.